contents/en/best-practice/chart-specificatio/bar/stacked-bar.md

```diff
@@ -0,0 +1,21 @@
+# Stacked Bar Chart
+
+The stacked bar chart is an extension of the bar chart. The difference is that the data in the bar chart is placed parallel while stacked together in the stacked bar chart. You can use a stacked bar chart to reflect the total amount of each category, as well as the proportional relationship of each data compared with the category. Therefore, it is very suitable for dealing with the relationship between part and whole.
+
+Compared with the pie chart, a stacked bar chart can display several groups of "part and whole" relationship. For instance, you can use a pie or bar chart to display the statistics of the number of people in each project in a physical education course. You can use a stacked bar chart when you want to distinguish boys and girls in each project. As shown in the chart, it included the info of the "part and whole" relationship of each group.
+
+<iframe max-width="830" width="100%" height="400" 
+ src="https://gallery.echartsjs.com/view-lite.html?cid=xBk7oUNwEz">
+</iframe>
+
+## Suggestion Using Stacked Bar Chart
```

Comment:
Maybe `Suggestion for Using Stacked Bar Chart` is better.

Reply:
I'll change rest of the title together.